galaxy_ng/app/api/v1/serializers.py

```diff
@@ -336,6 +342,40 @@ def get_download_count(self, obj):
         return 0
 
 
+class LegacyRoleRepositoryUpdateSerializer(serializers.Serializer):
+    name = serializers.CharField(required=False, allow_blank=False, max_length=50)
+    original_name = serializers.CharField(required=False, allow_blank=False, max_length=50)
```

Comment:
Should I be able to edit `original_name`?

Reply:
That's a good question. I think longterm, no you shouldn't be able to. For the short term though, we need to be able to set that based on the old api data.